packages/blockchain-link/src/types/events.ts

```diff
@@ -1,10 +1,10 @@
-interface ConnectEvent {
-    foo: boolean;
-}
+import { BlockEvent, NotificationEvent, FiatRatesEvent } from './responses';
 
 export interface Events {
-    connect: ConnectEvent;
-    disconnect: any;
-    notification: any;
-    block: any;
+    connected: undefined;
+    disconnected: undefined;
```

Comment:
`undefined` type?

Reply:
undefined function argument, this interface is a key/value mapping (event-type-name: event-callback-param)
https://github.com/trezor/trezor-suite/blob/2aa2d54f0125ca486d04281ad6289cf68d21796e/packages/blockchain-link/src/index.ts#L62

connect and disconnect doesnt send any additional data in emit:
https://github.com/trezor/trezor-suite/blob/2aa2d54f0125ca486d04281ad6289cf68d21796e/packages/blockchain-link/src/index.ts#L253

Reply:
btw this could be useful for typings in `desktopApi`